File: src/main/java/com/github/zly2006/reden/PreLaunch.java
package com.github.zly2006.reden;

import com.github.zly2006.reden.utils.UtilsKt;
import net.fabricmc.loader.api.entrypoint.PreLaunchEntrypoint;

public class PreLaunch implements PreLaunchEntrypoint {
    @Override
    public void onPreLaunch() {
        UtilsKt.checkMalilib();
    }
}

File: src/main/java/com/github/zly2006/reden/PreLaunch.kt
package com.github.zly2006.reden

import com.github.zly2006.reden.utils.checkMalilib
import net.fabricmc.loader.api.entrypoint.PreLaunchEntrypoint

class PreLaunch: PreLaunchEntrypoint {
    override fun onPreLaunch() {
        checkMalilib()
        System.setProperty("org.lwjgl.util.NoChecks", "true")
    }
}
